css/style/utility.css

```diff
@@ -34,6 +34,9 @@
 .evenly-padding-md{
     padding:1.5rem;
 }
+.padding-y{
+    padding:2rem 0;
+}
```

Comment:
I think it's better to write it explicitly 
```
.padding-y {
  padding-top: 2rem;
  padding-bottom: 2rem;
}
```
I had done something like what you have done. I bumped into a scenario where it was conflicting with padding-x. .